cli/internal/ffi/proto/messages.pb.go

```diff
@@ -144,7 +144,6 @@ type GlobResp struct {
 	unknownFields protoimpl.UnknownFields
 
 	// Types that are assignable to Response:
-	//
```

Comment:
huh?

Reply:
I'm having the same issue. I think our pre commit hooks are changing this file, which might indicate something's wrong in main.

Reply:
😮‍💨 https://vercel.slack.com/archives/C02HEJASXGD/p1675715378860089